packages/components/src/components/tabs/shadow.tsx

```diff
@@ -30,6 +32,15 @@ import type { Generic } from 'adopted-style-sheets';
 import { KolButtonGroupWcTag, KolButtonWcTag } from '../../core/component-names';
 // https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html
 
+enum Key {
```

Comment:
Wir nutzen diese Event-Namen auch an anderer Stelle, um Keyboard-Events zu behandeln. Ich würde vorschlagen, den Enum an einer zentralen Stelle abzulegen, zu exportieren und von dort zu verwenden. In einem neuen Ticket können wir dann die bestehenden Verwendungen refactoren.

Reply:
dann auch in diesem Ticket alles umsetzen. Mir ist auch noch nicht klar wo hin, schema ist es ja nicht und enums ist etwas komisch. Es fehlt eigentlich ein allgemeinen type ordner.